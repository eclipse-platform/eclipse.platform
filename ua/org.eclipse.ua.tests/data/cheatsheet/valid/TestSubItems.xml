<?xml version="1.0" encoding="UTF-8" ?> 
<!--
     Copyright (c) 2005, 2006 IBM Corporation and others.

     This program and the accompanying materials
     are made available under the terms of the Eclipse Public License 2.0
     which accompanies this distribution, and is available at
     https://www.eclipse.org/legal/epl-2.0/

     SPDX-License-Identifier: EPL-2.0
    
     Contributors:
         IBM Corporation - initial API and implementation
 -->

<cheatsheet title="Testing SubItems">

<intro 
	href="/org.eclipse.platform.doc.user/reference/ref-cheatsheets.htm">
	<description>
This cheat sheet will test subitems.
Let's get started!
	</description>
</intro>

<item
	title="Test SubItems"
	href="/org.eclipse.platform.doc.user/concepts/chelpsys.htm">
	<description>
Try the help link, you should see the concepts for the help system.
	</description>
	<subitem label="sub 1" skip="true"/>
	<subitem label="sub 2" skip="true">
		<action
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 3" skip="false"/>
	<subitem label="sub 4" skip="false">
		<action
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 5"/>
	<subitem label="sub 6">
		<action
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 7">
		<action
			confirm="true"
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 8">
		<action
			confirm="false"
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 9" skip="true">
		<action
			confirm="true"
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 10" skip="true">
		<action
			confirm="false"
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
	<subitem label="sub 11" skip="false">
		<action
			confirm="false"
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.NoopAction"/>
	</subitem>
	<subitem label="sub 12" skip="false">
		<action
			confirm="true"
			pluginId="org.eclipse.ua.tests"
			class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"/>
	</subitem>
</item>


<item
	title="All done!">
	<description>
Did the test work?
	</description>
</item>

</cheatsheet>
