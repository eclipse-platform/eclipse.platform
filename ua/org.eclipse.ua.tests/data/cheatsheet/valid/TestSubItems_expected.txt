<?xml version="1.0" encoding="UTF-8" ?>
<cheatsheet
      title="Testing SubItems">
   <item
         title="Introduction"
         description="This cheat sheet will test subitems.
Let's get started!"
         Href="/org.eclipse.platform.doc.user/reference/ref-cheatsheets.htm"
         contextId="null">
      <nullAction/>
      <nullList/>
      <nullPerformWhen/>
      <nullList/>
   </item>
   <list>
      <item
            title="Test SubItems"
            description="Try the help link, you should see the concepts for the help system."
            Href="/org.eclipse.platform.doc.user/concepts/chelpsys.htm"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <list>
            <subItem
                  label="sub 1"
                  when="null"
                  isSkip="true">
               <nullAction/>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 2"
                  when="null"
                  isSkip="true">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="false">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 3"
                  when="null"
                  isSkip="false">
               <nullAction/>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 4"
                  when="null"
                  isSkip="false">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="false">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 5"
                  when="null"
                  isSkip="false">
               <nullAction/>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 6"
                  when="null"
                  isSkip="false">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="false">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 7"
                  when="null"
                  isSkip="false">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="true">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 8"
                  when="null"
                  isSkip="false">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="false">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 9"
                  when="null"
                  isSkip="true">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="true">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 10"
                  when="null"
                  isSkip="true">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="false">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 11"
                  when="null"
                  isSkip="false">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.NoopAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="false">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
            <subItem
                  label="sub 12"
                  when="null"
                  isSkip="false">
               <action
                     class="org.eclipse.ua.tests.cheatsheet.util.IncrementVariableAction"
                     pluginId="org.eclipse.ua.tests"
                     when="null"
                     isConfirm="true">
                  <nullArray/>
               </action>
               <nullPerformWhen/>
            </subItem>
         </list>
      </item>
      <item
            title="All done!"
            description="Did the test work?"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
   </list>
</cheatsheet>
